priv/repo/migrations/20210124200209_add_discord_association_code_and_discord_id.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Safira.Repo.Migrations.AddDiscordAssociationCodeAndDiscordId do
+  use Ecto.Migration
+
+  def change do
+    alter table(:attendees) do
+      add :discord_association_code, :uuid, null: false
+      add :discord_id, :string
+    end
+
+      create unique_index(:attendees, [:discord_id])
```

Comment:
Isn't an uuid unique by default?

Reply:
"The probability that a UUID will be duplicated is not zero, it is close enough to zero to be negligible". Although this is not crucial (and probably everything will be just fine), you could do the verification, by introducing an unique_index and unique_constraint in changeset.

Reply:
I wrote two separate `create unique_index` statements because I'm not sure that if I write  `create unique_index(:attendees, [:discord_association_code, :discord_id])` it won't create a index compsed of both attributes together